pybossa/forms/forms.py

```diff
@@ -799,7 +799,7 @@ class UserPrefMetadataForm(Form):
         lazy_gettext('Additional comments'), default="")
     profile = TextAreaField(
         lazy_gettext('Profiles(json format)'), [is_json(dict)], default="",
-        render_kw={"placeholder": "{'finance': 0.5, 'art': 0.8}"})
+        render_kw={"placeholder": '{"finance": 0.5, "art": 0.8}'})
```

Comment:
Might it make sense to use a more realistic placeholder text here, like '{"english":0.8, "finance": 0.5}'?

Reply:
Sure.
 The reason I didn't put "English" is that, the language constraints are included in `user_pref.languages` so I don't want to give users impression that they need to discard the old user preference settings and put language constraints here. If you think it is not confusing, we can add English in sample text.

Reply:
Either way I'm sure its not a very big deal, but I would think that an English language requirement would be more of a hard constraint and something like art would be more of a preference. Up to you though!